### PR TITLE
test: add integration test for tumblr

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

File: tumblr-gif-search/index/_merge_matches_topk.yml [hunk 1]
```diff
@@ -16,7 +16,17 @@ requests:
       - !SliceQL
         with:
           start: 0
-          end: 10
+          end: 200
```
Comment (JoanFM) [inline tumblr-gif-search/index/_merge_matches_topk.yml:19]: why do we do this?

**Review by JoanFM — DISMISSED** [review]
Remember to add the `test` to the `CI`

File: tumblr-gif-search/index/_merge_matches_topk.yml (hunk 1 above)
Comment (cristianmtr) [reply tumblr-gif-search/index/_merge_matches_topk.yml]: mistake, reverted. Was testing something

**Review by nan-wang — CHANGES_REQUESTED** [review]
Let's keep the minimal number of test data because some of them are quite big

**Comment by cristianmtr** [comment]
> Let's keep the minimal number of test data because some of them are quite big

Reduced it to 3 samples and adjusted the TOPK

**Comment by cristianmtr** [comment]
> Remember to add the `test` to the `CI`

Done

File: tumblr-gif-search/tests/test_tumblr_integration.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+
+def get_flow():
+    f = Flow().load_config('flow-query.yml')
+    f.use_rest_gateway()
+    return f
+
+
+def test_query(tmpdir):
+    env_setup(tmpdir)
```
Comment (JoanFM) [inline tumblr-gif-search/tests/test_tumblr_integration.py:56]: env_setup could be a `fixture`

File: tumblr-gif-search/tests/test_tumblr_integration.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+SHARDS_CHUNK_SEG = 2
+SHARDS_INDEXER = 2
+JINA_TOPK = 2
+GIF_BLOB = 'tests/data/*.gif'
+NR_CASES_SEARCH = 3
+REQ_SIZE = 1
+
+
+@pytest.fixture()
+def env_setup(tmpdir):
```
Comment (cristianmtr) [inline tumblr-gif-search/tests/test_tumblr_integration.py:22]: @JoanFM Like this?

**Review by JoanFM — APPROVED** [review]

File: tumblr-gif-search/tests/test_tumblr_integration.py (hunk 3 above)
Comment (JoanFM) [reply tumblr-gif-search/tests/test_tumblr_integration.py:22]: Sure

**Pull request merged**